pyqtgraph/__init__.py

```diff
@@ -280,6 +280,15 @@ def renamePyc(startDir):
 from .Qt import isQObjectAlive
 from .ThreadsafeTimer import *
 
+# indirect imports used within library
+from .GraphicsScene import GraphicsScene
+from .python2_3 import asUnicode
+from .util.cupy_helper import getCupy
+
+# indirect imports known to be used outside of the library
+from .metaarray import MetaArray
+from collections import OrderedDict
```

Comment:
This import line will obscure the deprecation warning for anyone importing OrderedDict from pyqtgraph. We should keep it as `from .orderedict` so as to properly prepare such importers.

Reply:
I didn't think of that, so I tried it out. It seems that ```from collections``` yields the same behavior as current master whereas ```from .ordereddict``` obscures it.

current master
```
d:\>py -c "import pyqtgraph as pg; pg.OrderedDict"

d:\>py -c "from pyqtgraph.ordereddict import OrderedDict"
<string>:1: DeprecationWarning: OrderedDict is in the standard library for supported versions of Python. Will be removed in 0.13
```

with ```from collections import OrderedDict```
```
d:\>py -c "import pyqtgraph as pg; pg.OrderedDict"

d:\>py -c "from pyqtgraph.ordereddict import OrderedDict"
<string>:1: DeprecationWarning: OrderedDict is in the standard library for supported versions of Python. Will be removed in 0.13
```

with ```from .ordereddict import OrderedDict```
```
d:\>py -c "import pyqtgraph as pg; pg.OrderedDict"

d:\>py -c "from pyqtgraph.ordereddict import OrderedDict"
```

Reply:
Yeah, I realized that would be the case after I went to bed last night. I guess the better warning here would be to use `from .ordereddict` in __init__.py and replace ordereddict.py with:

```python
from collections import OrderedDict as _OrderedDict

import warnings


class OrderedDict(_OrderedDict):
    def __init__(self, *a, **kw):
        warnings.warn(
            "OrderedDict is in the standard library for supported versions of Python. Will be removed in 0.13",
            DeprecationWarning,
            stacklevel=2,
        )
        super(OrderedDict, self).__init__(*a, **kw)
```

Reply:
I'm :+1: for warnings in `__init__` method (it's what I did for MetaArray)